src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx

```diff
@@ -198,14 +211,16 @@ const ObjectDetailPage: NextPage = () => {
 					)}
 					<div className="p-object-detail__video">
 						{mediaType ? (
-							<FlowPlayer
-								className="p-object-detail__flowplayer"
-								src="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4"
-								poster="https://via.placeholder.com/1920x1080"
-								title="Elephants dream"
-								pause={pauseMedia}
-								onPlay={() => setPauseMedia(false)}
-							/>
+							!isLoadingPlayableUrl && playableUrl ? (
+								<FlowPlayer
+									className="p-object-detail__flowplayer"
+									src={playableUrl}
+									poster="https://via.placeholder.com/1920x1080"
+									title="Elephants dream"
```

Comment:
Title isn't correct anymore. Pass nothing or maybe the object name?

Reply:
Fixed in future pr